config/initializers/openapi.rb

```diff
@@ -5,7 +5,9 @@
 RSpec::OpenAPI.title = "Register API"
 
 RSpec::OpenAPI.path = lambda { |example|
-  if (match = example.file_path.match(%r{spec/requests/api/(v\d+\.\d+)/}))
-    "public/openapi/#{match[1]}.yaml"
+  if (match = example.file_path.match(%r{spec/requests/api/(v\d+\_\d+)/}))
+    version = match[1].gsub("_", ".")
+
+    "public/openapi/#{version}.yaml"
   end
```

Comment:
Not related to the scope of this PR but I don't think we should configure this gem in an initializer as it will load the configuration on every boot. I suggest to move the configuration to a `spec/support` module or in `spec_helper`

Reply:
It might not be in scope, but I think it does make sense and, as it's the only thing the initializer actually handles it would probably clean up the code a little.